examples/pytorch/gat/train_ppi.py

```diff
@@ -60,7 +60,7 @@ def main(args):
     g = train_dataset[0]
     n_classes = train_dataset.num_labels
     num_feats = g.ndata['feat'].shape[1]
-    g = g.to(device)
+    g = g.int().to(device)
```

Comment:
We still need to cast it to int32?

Reply:
I think so.

Reply:
Hmm how much was the performance gain?  It's OK if you didn't benchmark it though; I'm just curious.  We may need to benchmark it later after 0.5.  I could imagine that newcomers will definitely ask why `g.int()` is necessary and how much the benefit is.

Reply:
For GATConv there is no performance gain, for SAGEConv and GraphConv where cusparse is invoked the performance gap is around 30% and I think the gap would be reduced after we merge tvm kernel.